cg/apps/sequencing_metrics_parser/api.py

```diff
@@ -14,7 +14,7 @@ def create_sample_lane_sequencing_metrics_for_flow_cell(
     flow_cell_directory: Path, bcl_converter: str
 ) -> List[SampleLaneSequencingMetrics]:
     """Parse the sequencing metrics data for the correct demultiplexing software into the sequencing statistics model."""
-    if bcl_converter == BclConverter.BCLCONVERT:
+    if bcl_converter == BclConverter.BCLCONVERT or bcl_converter == BclConverter.DRAGEN:
```

Comment:
Removing the dragen enum field is out of scope for this PR, but getting rid of it would be nice 😅